### PR TITLE
Added Enum class for deploy test

### PR DESCRIPTION
**Pull request by alexbeatnik**
For better code readability, a class Running has been added.
This class using in deploy and destroy tests.

### CODE REVIEW

File: oct/tests/deployment/deploy_app.py [hunk 1]
```diff
@@ -2,14 +2,21 @@
...
 import os
+from enum import Enum
 from pyats.topology import Device
 from pyats.aetest import Testcase, test
 from pyats.utils.fileutils import FileUtils
 from requests.exceptions import ConnectTimeout
 from oct.tests import run_testcase
 
 
+class Running(Enum):
```
Comment (extsoft) [inline oct/tests/deployment/deploy_app.py]:
```suggestion
class Status(Enum):
```

File: oct/tests/deployment/deploy_app.py [hunk 2]
```diff
@@ -2,14 +2,21 @@
...
+    RUN = 0
```
Comment (extsoft) [inline oct/tests/deployment/deploy_app.py]:
```suggestion
    ON = 0
```

File: oct/tests/deployment/deploy_app.py [hunk 3]
```diff
@@ -2,14 +2,21 @@
...
+    NOT_RUN = 2
```
Comment (extsoft) [inline oct/tests/deployment/deploy_app.py]:
```suggestion
    OFF = 2
```

File: oct/tests/deployment/deploy_app.py [hunk 4]
```diff
@@ -18,12 +25,12 @@ def copy_file_to_server(device: Device) -> None:
     )
 
 
-def is_deploy_app(device: Device, lines: int) -> bool:
+def is_deploy_app(device: Device, is_run: Running) -> bool:
```
Comment (extsoft) [inline oct/tests/deployment/deploy_app.py]:
```suggestion
def has_status(device: Device, status: Starus) -> bool:
```

Comment (alexbeatnik) [reply oct/tests/deployment/deploy_app.py]: @extsoft Fixed

File: oct/tests/deployment/deploy_app.py [hunk 5]
```diff
@@ -18,12 +25,12 @@ def copy_file_to_server(device: Device) -> None:
...
 
 
-def is_deploy_app(device: Device, lines: int) -> bool:
+def is_deploy_app(device: Device, is_run: Status) -> bool:
     non_empty_lines_counter = 0
     for iterator in device.execute("docker-compose ps -q"):
         if iterator == "\n":
             non_empty_lines_counter += 1
-    return non_empty_lines_counter == lines
+    return non_empty_lines_counter == is_run
```
Comment (extsoft) [inline oct/tests/deployment/deploy_app.py]: @alexbeatnik this line always returns `False`.
See 
```python
>>> Status.ON == 2
False
>>> Status.ON == 0
False
>>> Status.ON.value == 0
True
```

Please update the code.

File: oct/tests/deployment/deploy_app.py [hunk 6]
```diff
@@ -2,14 +2,21 @@
...
+from enum import Enum
 from pyats.topology import Device
 from pyats.aetest import Testcase, test
 from pyats.utils.fileutils import FileUtils
 from requests.exceptions import ConnectTimeout
 from oct.tests import run_testcase
 
 
+class Status(Enum):
+    ON = 0
```
Comment (extsoft) [inline oct/tests/deployment/deploy_app.py]: @alexbeatnik I believe that if the app is working it has to be 
```suggestion
    ON = 2
```
and 0 for `OFF`.

File: oct/tests/deployment/deploy_app.py (hunk 5 above)
Comment (alexbeatnik) [reply oct/tests/deployment/deploy_app.py]: Fixed

File: oct/tests/deployment/deploy_app.py (hunk 6 above)
Comment (alexbeatnik) [reply oct/tests/deployment/deploy_app.py]: Fixed

File: .vagrant/machines/dp-151/virtualbox/action_provision [hunk 7]
```diff
@@ -0,0 +1 @@
+1.5:58e4edb4-147c-4208-8d18-e9cc0f52b23e
```
Comment (extsoft) [inline .vagrant/machines/dp-151/virtualbox/action_provision]: @alexbeatnik I believe that `.vagrant` has to be ignored, hasn't it? If yes, please add it `.gitignore`.

File: oct/tests/deployment/deploy_app.py [hunk 8]
```diff
@@ -35,7 +42,7 @@ def deploy_app(self, device: Device) -> None:
             copy_file_to_server(device)
             device.execute("cd oct")
             device.execute(f"APP_HOST={device.connections.main.ip} docker-compose up -d")
-            assert is_deploy_app(device, 2)
+            assert is_deploy_app(device, Status.ON.value)
```
Comment (extsoft) [inline oct/tests/deployment/deploy_app.py]: @alexbeatnik second argument should be a `Status`, not an `int`.

**Comment by extsoft** [comment]
Close as https://github.com/extsoft/dp-151/pull/82 is created.

**Pull request closed without merging**